api/graphql/application_errors.py

```diff
@@ -5,7 +5,34 @@
 
 
 class ValidationErrorCodes(Enum):
-    pass
+    RESERVATION_UNIT_NOT_RESERVABLE = "RESERVATION_UNIT_NOT_RESERVABLE"
```

Comment:
@kieferbonk @siren What do you think, are these error codes clear enough so that no extra documentation is not needed?